src/tab/ExNavigationTab.js

```diff
@@ -3,7 +3,7 @@
  */
 
 import React, { Children } from 'react';
-import { StyleSheet, View } from 'react-native';
+import { StyleSheet, View, Platform, Dimensions, PixelRatio } from 'react-native';
```

Comment:
Just noticed the unused variables here also.  Sorry for not catching the first time